root/components/home/Intro.js

```diff
@@ -0,0 +1,265 @@
+  {key: 1, label: 'Release'},
+  {key: 2, label: 'Recording'},
+  {key: 3, label: 'Label'},
+  {key: 4, label: 'Work'},
+  {key: 5, label: 'Release Group'},
+  {key: 6, label: 'Area'},
+  {key: 7, label: 'Place'},
+  {key: 8, label: 'Annotation'},
+  {key: 9, label: 'CD Stud'},
+  {key: 10, label: 'Editor'},
+  {key: 11, label: 'Tag'},
+  {key: 12, label: 'Instrument'},
+  {key: 13, label: 'Series'},
+  {key: 14, label: 'Event'},
+  {key: 15, label: 'Documentation'},
+];
+
+let typeCurrent = 'Artist';
+
+export default class Intro extends React.Component {
+    state = {
+      data: 'Actively looking for a barcode...',
+      show: false,
+    };
+
+    handleClose = () => {
+      this.setState({show: false});
+    }
+
+    handleShow = () => {
+      this.setState({show: true});
+    }
+
+    render() {
+      return (
+        <section className="intro d-flex align-items-center" id="intro">
+          <div className="container">
+            <div className="row">
+              <div
+                className="col-lg-9 d-flex flex-column"
+              >
+                <h1
+                  className="ms-4 mt-4"
+                  style={{color: 'white', height: '5vh'}}
+                >
+                  {l('The Music Database')}
+                </h1>
+                <h2
+                  className="ms-4 mt-2"
+                  style={{color: 'white', height: '5vh'}}
+                >
+                  {l(`World's Biggest Open Source Music Database`)}
+                </h2>
+                <div className="row mt-2 ms-2 mb-2">
+                  <div className="col-8 col-md-10">
+                    <input
+                      className="form-control special-font"
+                      id="searchInputMain"
+                      name="query"
+                      onKeyDown={event => {
+                        if (event.key === 'Enter') {
+                          const query =
+                              document.getElementById('searchInputMain');
+                          console.log(query.value);
+                          if (query.value.trim().length < 1) {
+                            return false;
+                          }
+                          let searchType;
+                          if (typeCurrent === 'CD Stub') {
+                            searchType = 'cdstub';
+                          } else if (typeCurrent === 'Documentation') {
+                            searchType = 'doc';
+                          } else {
+                            searchType =
+                                typeCurrent.replace(' ', '_').toLowerCase();
+                          }
+                          window.open('https://musicbrainz.org/' + 'search?type=' + searchType + '&query=' + query.value, '_self');
+                          return false;
+                        }
+                        return false;
+                      }}
+                      placeholder="Search 41,054,421 Entities"
```

Comment:
You'll of course want to actually make this placeholder translatable and the number of entities configurable :) (I know is a draft, but just so we don't forget).

Reply:
Makes sense. Thanks!